newIDE/app/src/PropertiesEditor/PropertiesMapToSchema.js

```diff
@@ -18,12 +18,21 @@ export default (
     propertyName: string,
     newValue: string
   ) => void,
-  object: ?gdObject
+  object: ?gdObject,
+  groupFilter: ?string
 ): Schema => {
   const propertyNames = properties.keys();
   const propertyFields = mapFor(0, propertyNames.size(), i => {
     const name = propertyNames.at(i);
     const property = properties.get(name);
+    if (groupFilter || groupFilter === '') {
```

Comment:
If we use the approach of doing the grouping here, by putting fields into "container" fields with titles, we avoid doing N*M iterations, N being the number of properties and M the number of groups. Not a big deal, but when rendering large list of properties, this can start to be visible on low end devices I think.